g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/CLDRUtils.java

```diff
@@ -1125,8 +1128,63 @@ public static void languageDataExtract() {
         logger.info("Extract languageData.json data complete!");
     }
 
+    
+    public static void writeJsonStr2File(String filePath, String jsonStr) {
```

Comment:
writeJsonStr2File and writePatternDataIntoFile may be unified into one method, or you can directly use writePatternDataIntoFile

Reply:
can't use it